ntoskrnl/kdbg/kdb_cli.c

```diff
@@ -2109,7 +2113,9 @@ KdbpCmdMod(
     KdbpPrint("  Base      Size      Name\n");
     for (;;)
     {
-        KdbpPrint("  %08x  %08x  %wZ\n", LdrEntry->DllBase, LdrEntry->SizeOfImage, &LdrEntry->BaseDllName);
+        KdbpPrint("  %p  %08x  ", LdrEntry->DllBase, LdrEntry->SizeOfImage);
+        KdbpPrintUnicodeString(&LdrEntry->BaseDllName);
+        KdbpPrint("\n");
```

Comment:
I wonder if it's possible to patch `KdbpPrint` function itself to check for IRQL level there and use correct Unicode printing function (to avoid separating function calls here).

Reply:
The `KdbpPrint` function uses the CRT `_vsnprintf` function to perform the main formatting, and it is this latter one that deals e.g. with the `%wZ` formatter.